server/src/main/java/org/candlepin/util/X509V3ExtensionUtil.java

```diff
@@ -77,10 +80,12 @@
     private static final Object END_NODE = new Object();
     private static boolean treeDebug = false;
     @Inject
-    public X509V3ExtensionUtil(Configuration config, EntitlementCurator entCurator) {
+    public X509V3ExtensionUtil(Configuration config, EntitlementCurator entCurator,
+            ProductServiceAdapter prodAdapter) {
```

Comment:
I fear we should hit the V1 extension util as well, I can't think of a good reason not too.

We really need to get these paths for cert generation refactored into something a little more refined sometime but that can wait.